src/components/shared/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+// material ui
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+import CheckIcon from '@material-ui/icons/Check';
+import ClearIcon from '@material-ui/icons/Clear';
+import { green } from '@material-ui/core/colors';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+export interface SimpleDialogProps {
+    title: string;
+    isOpen: boolean;
+    handleClose: (value: boolean) => void;
+}
+
+export function ConfirmationDialog({ handleClose, isOpen, title }: SimpleDialogProps) {
+  
+    return (
+      <Dialog onClose={() => handleClose(!open)} open={isOpen}>
```

Comment:
@showduhtung open is actually a function built into Dialog from Material UI and since this line is directly from the example, I feel like I should leave it.. what do you think?